dockers/docker-orchagent/start.sh

```diff
@@ -35,14 +35,14 @@ SWSSCONFIG_ARGS="00-copp.config.json "
 
 if [ "$HWSKU" == "Force10-S6000" ]; then
     ORCHAGENT_ARGS+="-m $MAC_ADDRESS"
-    SWSSCONFIG_ARGS+="td2.32ports.qos.json td2.32ports.buffers.json "
+    SWSSCONFIG_ARGS+="td2.32ports.buffers.json td2.32ports.qos.json "
```

Comment:
I want to know more information about this. As I don't see any failures while applying the configurations, why the order of applying these files matters? Are there any dependencies in between some SAI objects? How to ensure the order? I think it is better also to fix the dependency in orchagent code.